staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -304,6 +298,74 @@ func (w *WebhookAuthorizer) Authorize(ctx context.Context, attr authorizer.Attri
 
 }
 
+func resourceAttributesFrom(attr authorizer.Attributes) *authorizationv1.ResourceAttributes {
+	ret := &authorizationv1.ResourceAttributes{
+		Namespace:   attr.GetNamespace(),
+		Verb:        attr.GetVerb(),
+		Group:       attr.GetAPIGroup(),
+		Version:     attr.GetAPIVersion(),
+		Resource:    attr.GetResource(),
+		Subresource: attr.GetSubresource(),
+		Name:        attr.GetName(),
+	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.AuthorizeWithSelectors) {
+		// If there's an error while parsing selectors, we can safely ignore the selector
+		// This is because selectors only narrow, so if broader permission is available, the request can be allowed.
+		if selector, selectorErr := attr.ParseFieldSelector(); selectorErr == nil && selector != nil {
+			retRequirements := []authorizationv1.FieldSelectorRequirement{}
+			for _, requirement := range selector.Requirements() {
+				retRequirement := authorizationv1.FieldSelectorRequirement{}
+				switch {
+				case requirement.Operator == selection.Equals || requirement.Operator == selection.DoubleEquals || requirement.Operator == selection.In:
+					retRequirement.Operator = metav1.LabelSelectorOpIn
+					retRequirement.Key = requirement.Field
+					retRequirement.Values = []string{requirement.Value}
+				case requirement.Operator == selection.NotEquals || requirement.Operator == selection.NotIn:
+					retRequirement.Operator = metav1.LabelSelectorOpIn
```

Comment:
copy / paste that reverses the logic :)

add a unit test that would catch this?